src/transformers/models/deit/modeling_deit.py

```diff
@@ -120,10 +171,10 @@ def forward(self, pixel_values: torch.Tensor) -> torch.Tensor:
             raise ValueError(
                 "Make sure that the channel dimension of the pixel values match with the one set in the configuration."
             )
-        if height != self.image_size[0] or width != self.image_size[1]:
-            raise ValueError(
-                f"Input image size ({height}*{width}) doesn't match model ({self.image_size[0]}*{self.image_size[1]})."
-            )
+        # if height != self.image_size[0] or width != self.image_size[1]:
+        #     raise ValueError(
+        #         f"Input image size ({height}*{width}) doesn't match model ({self.image_size[0]}*{self.image_size[1]})."
+        #     )
```

Comment:
We shouldn't add commented out code

Reply:
Should I keep this check if, interpolate_pos_encoding is set to False?

Reply:
I would just remove the check - we shouldn't need to pass in `interpolate_pos_encoding` to the patch embeddings class